docker-machine-nfs.sh

```diff
@@ -244,4 +244,4 @@ restartBoot2Docker
 
 verifyNFSMount
 
-showFinish
```

Comment:
I think the removal of the whitespace here was done by my editor and not an intentional change on my part.